package.json

```diff
@@ -27,6 +27,12 @@
     "sequelize"
   ],
   "author": "Mick Hansen <maker@mhansen.io>",
+  "contributors": [
+    {
+      "name": "Glavin Wiechert",
+      "url": "https://github.com/Glavin001"
```

Comment:
It may make more sense when adding a contributors key to make the name `graphql-sequelize community` and the url `https://github.com/mickhansen/graphql-sequelize/graphs/contributors` instead of individually name every person who contributes a pull request.